### PR TITLE
feat: upload certificates and public keys for verifying signatures

### PR DESCRIPTION
**Pull request by Andreea-Lupu**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — APPROVED** [review]
LGTM

**Review by andaaron — APPROVED** [review]

**Comment by rchincha** [comment]
@Andreea-Lupu this looks good, let's demo this and merge.

File: pkg/meta/signatures/config_verifying_signatures.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Verifying signatures
```
Comment (rchincha) [inline pkg/meta/signatures/config_verifying_signatures.md:1]: call this file README.md

**Comment by rchincha** [comment]
Rationale behind the URL endpoint scheme in the commit msg as well as inline comments.

**Review by andaaron — APPROVED** [review]

**Pull request merged**